src/main/java/org/example/Cocktail.java

```diff
@@ -0,0 +1,72 @@
+package org.example;
+
+public class Cocktail {
+    private final String base;
+    private final String mixer;
+    private final String sweetener;
+    private final String beauty;
+    private final int iceCount;
+
+    private Cocktail(final CocktailBuilder cocktailBuilder) {
```

Comment:
It's better that Cocktail doesn't know about the builder to avoid tightly coupling.

Reply:
Done